### PR TITLE
Update dependencies

### LINKED ISSUE #50: Update dependencies
The used version of `packaging` is old and conflicts with updating in my other dependency.

```
  Because poetryup (0.7.1) depends on packaging (>=21.3,<22.0)
   and fs-gcsfs (1.4.5) depends on packaging (>=20.0,<21.0), poetryup (0.7.1) is incompatible with fs-gcsfs (1.4.5).
  And because no versions of fs-gcsfs match >1.4.5,<2.0.0, poetryup (0.7.1) is incompatible with fs-gcsfs (>=1.4.5,<2.0.0).
  So, because *** depends on both fs-gcsfs (^1.4.5) and poetryup (0.7.1), version solving failed.

```

### PR DESCRIPTION
**Pull request by mathiaseitz**
Fixes #50

### CODE REVIEW

**Review by ronaldrichter — APPROVED** [review]

**Review by markounikau — APPROVED** [review]

**Review by trendelkampschroer — COMMENTED** [review]
I had a PR open with matching changes that never got a green light from CI, cf. https://github.com/Othoz/gcsfs/pull/49. Did you change anything in the build pipeline in the meantime?

**Comment by mathiaseitz** [comment]
@trendelkampschroer I am now working on getting Travis running again. Will ping you once this is done so we can merge your PR first.

**Pull request merged**
